src/features/organizations/components/OrganizationTree.tsx

```diff
@@ -16,7 +16,11 @@ interface OrganizationTreeProps {
 
 function renderTree(props: OrganizationTreeProps): React.ReactNode {
   const { treeItemData, orgId, onSwitchOrg } = props;
-  return treeItemData.map((item) => (
+  const sortedTreeItem = [...treeItemData].sort((a, b) =>
```

Comment:
I think this variable name should be plural, to indicate that it's a list/array of items, i.e. `sortedTreeItems`.